tools/indexes-creator/cmd/indexes-creator/config/cluster.toml

```diff
@@ -0,0 +1,6 @@
+[config]
+    url = "http://localhost:9200"
```

Comment:
align all items in regards to whitespaces

Reply:
done.